pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -6499,7 +6499,6 @@ broken-packages:
   - hslogger-template
   - hslogstash
   - hsluv-haskell
-  - hsmagick
```

Comment:
This line being here marks `hsmagick` as broken.  This means that when `hackage2nix` runs the next time and regenerates the `pkgs/development/haskell-modules/hackage-packages.nix` file, the `hsmagick` derivation will have `broken = true` set.

However, if you remove this `hsmagick` line here, then the next time `hackage2nix` is run, it will generate the hsmagick` derivation without `broken = true`, effectively marking it unbroken.

As it is, I don't think this PR is necessary.

-----------------------------------------------

Based on the title of this PR, it sounds like you want to completely stop the `haskellPackages.hsmagick` derivation from ever getting created.

I'm not sure that this functionality exists in the hackage2nix tooling.  In general, we expose all packages available from Hackage.  However, we leave a lot of the packages just marked `broken`.  It looks like `hsmagick` is currently marked broken, so I think the current situation should be alright, and this PR can probably be closed?

Reply:
Yeah, I wasn't sure about that. I guessing because it's `broken` it won't eagerly evaluate jasper missing?

Reply:
When evaluating nixpkgs, I don't _think_ it will throw an error for jasper missing.  If it does throw an error, I imagine it will get fixed up before `haskell-updates` is merged into `master`.

Reply:
Gotcha, thanks.